### DISCUSSION TITLE
I might be stoopid but the finnish layout won't compile

### DISCUSSION THREAD

**Discussion by Sweets-omg** [ArminJo/DigistumpArduino#33] [Q&A] [answered]
Like the title says, I'm a newbie but when trying to compile with the #define LAYOUT_FINNISH it spits out an error I have no clue of. I tested the default language, canadian french and german and all of those compiled fine. I hope the screenshots will give enough info, but I can share more if needed.
(edit: attached the screenshots instead of posting them as links)
(edit2: removed a accidental g that slipped in XD)
(edit3: Tested versions 1.7.5, 1.7.4, 1.7.3)
![64OkNuawtN](https://user-images.githubusercontent.com/55176185/150411132-949ff24b-cc8c-4c07-a059-f7be4308d900.png)
![javaw_i2Ys0swtO7](https://user-images.githubusercontent.com/55176185/150411149-bc40366a-2472-4da0-8baf-bb8661a7bd52.png)

**Comment by ArminJo** [comment, marked as answer]
I opened an issue for this.
Thanks for reporting!
Please replace your keyboard.h with https://github.com/ArminJo/DigistumpArduino/blob/master/digistump-avr/libraries/DigisparkKeyboard/keylayouts.h
